interface/usergroup/usergroup_admin_add.php

```diff
@@ -261,7 +261,9 @@ function authorized_clicked() {
   <?php echo generate_select_list("erxrole", "newcrop_erx_role", '','','--Select Role--','','','',array('style'=>'width:120px')); ?>
 </td>
 </tr>
-
+<tr>
+<td><span class="text"><?php xl('Weno Provider ID','e'); ?>: </span></td><td><input type="text" name="erxprid" style="width:120px;"  value="<?php echo $iter["weno_prov_id"]?>"></td>
```

Comment:
Since new line, need to follow current security model for html escaping
``````
<td><span class="text"><?php echo xlt('Weno Provider ID'); ?>: </span></td><td><input type="text" name="erxprid" style="width:120px;"  value="<?php echo attr($iter["weno_prov_id"]); ?>"></td>